test/integration/ComputeFlow.test.ts

```diff
@@ -259,7 +281,7 @@ describe('Simple compute tests', async () => {
     const txidAlgo = await datatoken.startOrder(
       datatokenAddressAlgo,
       consumerAccount,
-      initializeDataAlgo.computeAddress,
+      computeConsumerAddress, // this is important
```

Comment:
ok, maybe mention why it is important? Every param when required is important so to speak

Reply:
done